src/ReactErrorBoundary.re

```diff
@@ -0,0 +1,45 @@
+type reactComponentClass;
+
+[@bs.module "react"] external component: reactComponentClass = "Component";
```

Comment:
This is to ensure the `React` import matches the `bsconfig` spec for the project.